packages/aws-cdk-lib/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -295,6 +308,11 @@ export abstract class QueueProcessingServiceBase extends Construct {
    */
   public readonly scalingSteps: ScalingInterval[];
 
+  /**
+   * Grace period after scaling activity.
+   */
+  readonly cooldown?: Duration;
```

Comment:
```suggestion
  private readonly cooldown?: Duration;
```

Here, we want to be explicit about scope, i.e. public or private. I tend to always default to private unless there is a clear use case for people who want to reference the prop. In this case, there is not. We can always make it public later, but we can't go back and make it private. 

And then, since its private, we don't need any docstring at all :)